lib/node_modules/@stdlib/regexp/to-json/README.md

```diff
@@ -0,0 +1,170 @@
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var toJSON = require( '@stdlib/regexp/to-json' );
+```
+
+#### toJSON( regexp )
+
+Returns a [JSON][json] representation of a [regular expression][regexp] object.
+
+```javascript
+var regex = new RegExp('ab+c');
+
+var json = toJSON( regex );
+/* returns
+    {
+        'type': 'regexp',
+        'pattern': 'ab+c',
+        'flags': '...' // if present
+    }
+*/
+```
+
+The [JSON][json] `object` is **guaranteed** to have the following properties:
+
+-   **type**: regular expression type.
+-   **pattern**: regular expression string pattern.
+
+Depending on the object, the following property **may** be present:
+
+-   **flags**: aditional options for matching.
+
+The function also serializes **all** `enumerable` properties.
+
+<!-- eslint-disable object-curly-newline -->
+
+```javascript
+var regex = new RegExp('ab+c');
+regex.a = 'b';
+regex.c = { 'd': 'e' };
+
+var json = toJSON( regex );
+/* returns
+    {
+        'type': 'regexp',
+        'pattern': 'ab+c',
+        'flags': '...', // if present
+        'a': 'b',
+        'c': {
+            'd': 'e'
+        }
+    }
+*/
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "regexp" -->
+
+```javascript
+var toJSON = require( '@stdlib/regexp/to-json' );
+
+var regex = new RegExp('ab+c');
+var out = toJSON( regex );
+/* returns
+    {
+        'type': 'regexp',
+        'pattern': 'ab+c',
+        'flags': ''
+    }
+*/
+
+var regexf = new RegExp('ab+c', 'd');
+var out = toJSON( regexf );
+/* returns
+    {
+        'type': 'regexp',
+        'pattern': 'ab+c',
+        'flags': 'd'
+    }
+*/
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section to include cited references. If references are included, add a horizontal rule *before* the section. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="references">
+
+</section>
+
+<!-- /.references -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/regexp/regexp`][@stdlib/regexp/regexp]</span><span class="delimiter">: </span><span class="description">regular expression to parse a string.</span>
+
+</section>
+
+<!-- /.related -->
+
+<!-- Section for all links. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="links">
+
+[json]: http://www.json.org/
+
+[regexp]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+
+<!-- <related-links> -->
+
+[@stdlib/error/reviver]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/error/reviver
+
+<!-- </related-links> -->
+
```

Comment:
```suggestion
```

Related links are also automatically populated. So can remove. This link is also incorrect and should have been picked up by the Markdown linter.